app/controllers/catalog_controller.rb

```diff
@@ -43,6 +43,8 @@ def self.modified_field
     #   The ordering of the field names is the order of the display
     config.add_facet_field solr_name("human_readable_type", :facetable), label: "Type", limit: 5
     config.add_facet_field solr_name("resource_type", :facetable), label: "Resource Type", limit: 5
+    config.add_facet_field solr_name("nested_geo_points_label", :symbol), label: "Geographic Point", limit: 5
```

Comment:
Both of these should not be faceted, they'll be user-inputted labels and the chances of having more than one occurrence of the same value are slim.